packages/rocketchat-models/server/models/Messages.js

```diff
@@ -171,7 +171,7 @@ export class Messages extends Base {
 			query.pinned = { $ne: true };
 		}
 
-		if (!ignoreThreads) {
+		if (ignoreThreads) {
```

Comment:
So this should prevent the message starting the thread from being pruned if they want to keep threads?

Reply:
yes